packages/adblocker/src/filters/network.ts

```diff
@@ -38,6 +38,8 @@ import {
   findLastIndexOfUnescapedCharacter,
 } from '../utils.js';
 import IFilter from './interface.js';
+import { HTMLModifier } from '../html-filtering.js';
+import CosmeticFilter from './cosmetic.js';
```

Comment:
Nit. Maybe enough (not sure if it works) to import as type since it's only used in a type guard.
```suggestion
import type CosmeticFilter from './cosmetic.js';
```